packages/hoppscotch-common/src/components/environments/teams/index.vue

```diff
@@ -31,41 +31,57 @@
           v-if="team !== undefined && team.myRole === 'VIEWER'"
           v-tippy="{ theme: 'tooltip' }"
           disabled
-          :icon="IconArchive"
+          :icon="IconImport"
           :title="t('modal.import_export')"
         />
         <HoppButtonSecondary
           v-else
           v-tippy="{ theme: 'tooltip' }"
-          :icon="IconArchive"
+          :icon="IconImport"
           :title="t('modal.import_export')"
           @click="displayModalImportExport(true)"
         />
       </div>
     </div>
-    <HoppSmartPlaceholder
+    <div
       v-if="!loading && teamEnvironments.length === 0 && !adapterError"
-      :src="`/images/states/${colorMode.value}/blockchain.svg`"
-      :alt="`${t('empty.environments')}`"
-      :text="t('empty.environments')"
+      class="p-5 space-y-25 flex flex-col"
```

Comment:
```suggestion
      class="p-5 space-y-25"
```

Are the flex classes necessary here?

Reply:
Yes.

Reply:
same